roles/test_alerts/tasks/first_test.yml

```diff
@@ -0,0 +1,31 @@
+---
+# Assuming we've already logged in...
+# Following procedure on https://infrawatch.github.io/documentation/#creating-an-alert-rule-in-prometheus_assembly-advanced-features
+# Assuming we're in teh right project already...
+
+- name: "Create the alert"
+  shell:
+    cmd: |
+      oc apply -f - <<EOF
+      apiVersion: monitoring.coreos.com/v1
+      kind: PrometheusRule
+      metadata:
+        creationTimestamp: null
+        labels:
+          prometheus: default
+          role: alert-rules
+        name: prometheus-alarm-rules
+        namespace: service-telemetry
+      spec:
+        groups:
+          - name: ./openstack.rules
+            rules:
+              - alert: Collectd metrics receive rate is zero
+                expr: rate(sg_total_collectd_msg_received_count[1m]) == 0
+      EOF
+
+- name: "Check that the alert was created"
+  shell:
+    cmd: |
+      curl -k --user "{{ prom_user }}:{{ prom_pass }}" https://{{ prom_host }}/api/v1/rules
+  register: output
```

Comment:
WIP? I guess I don't see this called from anywhere yet either

Reply:
This is what I demoed on Monday. I am testing with an existing environment, and figuring out what needs to go where for set-up i.e. What is generic enough to go here, and what is something that will be passed as vars  by the calling playbook.
I plan to add some playbooks in the collection as a reference to run each role.